src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,582 @@
+                                            lat_c, lon_c, h_inc_ant,
+                                            indoor_cbsd,
+                                            reliability=reliabilities,
+                                            freq_mhz=FREQ_PROP_MODEL)
+    path_loss = np.array(results.db_loss)
+
+    # Compute EIRP of CBSD grant inside the frequency range of protection constraint
+    if dpa_type != 'out-of-band inland': # For co-channel offshore/inland DPAs
+
+        # CBSD co-channel bandwidth overlapping with
+        # frequency range of protection constraint
+        co_channel_bw = min(high_freq_cbsd, high_freq_c) - max(low_freq_cbsd, low_freq_c)
+
+        # Overlapping bandwidth between the first 10 MHz of CBSD grant adjacent-channel and
+        # the frequency range of protection constraint
+        adj_channel_bw = min(10000000.0, high_freq_c - low_freq_c - co_channel_bw)
+
+        if adj_channel_bw == 0:    # Consider only co-channel bandwidth
+            eirpPerMHz_cbsd = antenna.GetStandardAntennaGains(results.incidence_angles.hor_cbsd,
+                                                              ant_az_cbsd, ant_beamwidth_cbsd,
+                                                              maxEirpPerMHz_cbsd)
+            eirp_cbsd = eirpPerMHz_cbsd + 10 * np.log10(co_channel_bw / 1000000.0)
+
+        else:   # Consider both co-channel bandwidth and first 10 MHz adjacent-bandwidth
+            # Compute CBSD antenna gain in the direction of protection point
+            ant_gain = antenna.GetStandardAntennaGains(results.incidence_angles.hor_cbsd,
+                                                       ant_az_cbsd, ant_beamwidth_cbsd,
+                                                       ant_gain_cbsd)
+
+            # Compute conducted CBSD power within co-channel bandwidth
+            power_co_channel_bw = (maxEirpPerMHz_cbsd - ant_gain_cbsd) \
+                                   + 10 * np.log10(co_channel_bw / 1000000.0)
+
+            # Compute conducted CBSD power within adjacent-channel bandwidth
+            power_adj_channel_bw = OOB_POWER_WITHIN_10MHZ \
+                                   + 10 * np.log10(adj_channel_bw / 1000000.0)
+
+            # Compute total EIRP
+            power_cbsd_mW = np.power(10.0, power_co_channel_bw / 10.0)\
+                            + np.power(10.0, power_adj_channel_bw / 10.0)
+            eirp_cbsd = 10 * np.log10(power_cbsd_mW) + ant_gain
+
+    else:   # For out-of-band inland DPAs
+        # Compute CBSD antenna gain in the direction of protection point
+        ant_gain = antenna.GetStandardAntennaGains(results.incidence_angles.hor_cbsd,
+                                                   ant_az_cbsd, ant_beamwidth_cbsd,
+                                                   ant_gain_cbsd)
+
+        # Compute out-of-band conducted power
+        oob_power = ComputeOOBConductedPower(low_freq_cbsd, low_freq_c, high_freq_c)
```

Comment:
Here also, the oob_power is missing the overall power scalor of the cbsd, I believe. The ComputeOOBConductedPower actually computes a relative scalor, not a final power.